Fix side panel exit animation (#350)
seek-oss/playroom: .changeset/beige-tips-swim.md

---
'playroom': patch
---

> Fix minor issue in the side panel exit animation, preventing the contents from vanishing abruptly.

Reviewer: Not sure if you were aiming to describe the problem or the result, but you kind of ended up with both?
```suggestion
Fixes a bug in the side panel exit animation that was causing the contents to vanish abruptly
```
